src/components/Composer/implementation/index.tsx

```diff
@@ -241,6 +241,11 @@ function Composer(
                 e.preventDefault();
                 return;
             }
+
+            // Only prevent scrolling when composer is scrollable
```

Comment:
I find this comment confusing. Could we update it to explain more of the "why" the code is doing what it is doing here.

Reply:
Looks great.